client/stylesheets/_toCat.scss

```diff
@@ -2469,16 +2470,15 @@ i:hover {
 // }
 
 .headerSearch {
-    margin-top: 7px !important;
-    margin-bottom: 8px !important;
+    margin-top: 10px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered float, margin-top, vertical-align